nexus-sdk-js/src/views/SparqlView/index.ts

```diff
@@ -13,19 +13,25 @@ export interface SparqlViewResponse {
   _deprecated: boolean;
 }
 
+// based on: https://www.w3.org/TR/sparql11-results-json/
+// describes a SPARQL Query Results JSON
 interface SparqlViewQueryResponse {
   head: {
     vars: string[];
+    link?: string | string[];
   };
-  results: {
+  results?: {
     bindings: {
       [key: string]: {
-        datatype?: string;
         type: string;
         value: any;
+        datatype?: string;
```

Comment:
Works as is, but do we need to specify some common additional optional attributes like "datatype" here? Or just keep the any attribute bits?

Reply:
not sure I understand the question? Looks like the RFC says we always have `type`, `value` and maybe `datatype`, `xml:lang`. Then, there can be custom stuff. The only thing we cannot express with typescript is the fact that it is either `datatype` or `xml:lang`, but not both at the same time. But I don't think it's too much of an issue

Reply:
Indeed, after re-reading the RFC it seems it is the case. Ok to keep it as is.